searx/engines/seekr.py

```diff
@@ -0,0 +1,104 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# lint: pylint
+"""Seekr (images, videos, news)
+"""
+
+from datetime import datetime
+from json import loads
+from urllib.parse import urlencode
+
+about = {
+    "website": 'https://seekr.com/',
+    "official_api_documentation": None,
+    "use_official_api": True,
```

Comment:
When the API is official but not documented, from where did you got your get your know how the details from the API .. reversed engendered?

Reply:
Not sure if the API is official, I didn't search deeper about it - I reverse engineered the API requests in the browser dev tools.
From my research now the API is probably not official, I just thought so because we need to provide an `apiKey`, which is always the same when using it in the browser however.

Reply:
> I reverse engineered the API requests in the browser dev tools.

This is what I also often do :) .. TBH I haven't thought in deep, but I think the intention the flag `use_official_api` is to show if the engine uses an official API that is documented (and versioned) .. on the opposite we have engines where we know there is an  API but we didn't use the API:

https://github.com/searxng/searxng/blob/c76c999a1eeecae265573b12e81ae74f56988820/searx/engines/google.py#L42-L43

Even the google engine is reversed engineered, it can help to understand the arguments of the API when we follow (more or less) the official API.

I will change the value of the flag in my review ..